frontend/src/scenes/surveys/SurveyRepeatSchedule.tsx

```diff
@@ -22,29 +24,35 @@ function SurveyIterationOptions(): JSX.Element {
         <>
             <LemonField.Pure>
                 <LemonRadio
-                    value={schedule}
+                    value={survey.schedule}
                     onChange={(newValue) => {
-                        setSchedule(newValue as ScheduleType)
-                        if (newValue === 'once') {
+                        setSurveyValue('schedule', newValue)
+                        if (newValue === SurveySchedule.Once || newValue === SurveySchedule.Always) {
                             setSurveyValue('iteration_count', 0)
                             setSurveyValue('iteration_frequency_days', 0)
-                        } else if (newValue === 'recurring') {
+                        } else if (newValue === SurveySchedule.Recurring) {
                             setSurveyValue('iteration_count', 1)
                             setSurveyValue('iteration_frequency_days', 90)
                         }
                     }}
                     options={[
                         {
-                            value: 'once',
+                            value: SurveySchedule.Once,
                             label: 'Once',
                             'data-attr': 'survey-iteration-frequency-days',
                         },
                         {
-                            value: 'recurring',
+                            value: SurveySchedule.Recurring,
                             label: 'Repeat on a schedule',
                             'data-attr': 'survey-iteration-frequency-days',
                             disabledReason: surveysRecurringScheduleDisabledReason,
                         },
+                        {
+                            value: SurveySchedule.Always,
+                            label: 'All the time',
+                            'data-attr': 'survey-iteration-frequency-days',
+                            disabledReason: survey.type !== 'widget' ? 'Only available for widget surveys' : undefined,
```

Comment:
do we have an enum type for `widget` as well?

Reply:
changed, ty for review